src/routes/travelRoute.js

```diff
@@ -1,28 +1,21 @@
 import { Router } from 'express';
```

Comment:
You did not add a line of code in this file. Please, revert the changes.

Reply:
@ChidiChuks  please tidy up the commented lines